sema/bench/core.py

```
@@ -10,12 +10,24 @@
 
 log = getLogger(__name__)
 
 
+class SembenchResult(ServiceResult):
+    """Result of the sembench service"""
+
+    def __init__(self):
+        self._success = False
+
+    @property
+    def success(self) -> bool:
```

Ruff

Missing type annotation for `self` in method

(ANN101)